api/admin/controller/collection_settings.py

```diff
@@ -93,15 +95,25 @@ def load_settings(self, protocol_settings, collection_object, collection_setting
 
         settings = {}
         for protocol_setting in protocol_settings:
-            key = protocol_setting.get("key")
-            if not collection_settings or key not in collection_settings:
-                if key == 'mirror_integration_id':
-                    value = collection_object.mirror_integration_id or self.NO_MIRROR_INTEGRATION
-                elif protocol_setting.get("type") == "list":
-                    value = collection_object.external_integration.setting(key).json_value
-                else:
-                    value = collection_object.external_integration.setting(key).value
-                settings[key] = value
+            if protocol_setting:
```

Comment:
This isn't wrong, but you can make a diff like this a lot smaller and easier to read by doing it this way:

```
if not protocol_setting:
    continue
[all the other stuff]
```

That way you avoid adding an extra indent.